pkg/openapi/validation_test.go

```diff
@@ -86,6 +86,7 @@ func Test_matchGVK(t *testing.T) {
 		gvk            string
 		match          bool
 	}{
+
 		{
```

Comment:
Can you add another Ingress CRD to the test?

Reply:
I mean the CRD:
```
ingresses                                    config.openshift.io/v1                   false        Ingress
```

Reply:
yes. I'm trying to install openshift.

Reply:
@vyankyGH perhaps @realshuting means to add another test case such as:
```
		{
			"io.openshift.config.v1.Ingress",
			"networking.k8s.io/v1/Ingress",
			false,
		},
```